### ISSUE TITLE
Listen, why is the acc visible to everyone? they can control it

### ISSUE THREAD

**Issue by mrranger** [dilawfird/steambotauto#1]
Listen, why is the acc visible to everyone? they can control it

**Comment by dilawfird** [comment]
They can't control it but I will improve this soon why not adding a password to access the page, I will also make a little video showing how you can deploy it on heroku.
For the moment your can't delete your account when you add it but I will update it soon.

**Comment by mrranger** [comment]
That is, they see the account, but can’t do anything? will not be able to change the game

**Comment by mrranger** [comment]
You are the best, and where it is possible to offer me your options, ideas

**Comment by dilawfird** [comment]
you can create a issue by idea if you want

**Comment by mrranger** [comment]
I don’t understand why the bot works for only 1 hour, then all accounts are deleted

**Comment by dilawfird** [comment]
Because with the free plan, if your heroku app receives no web traffic in a 30 mins period it will sleep the application, I will fix it by making a request every 10 mins to the app.

**Comment by mrranger** [comment]
JavaScript

var http = require("http");
setInterval(function() {
    http.get("http://<your app name>.herokuapp.com");
}, 300000); // every 5 minutes (300000)

**Comment by dilawfird** [comment]
yeah I think with a simple get request like this, it should be fixed, I will update this to night and adding the possiblity to remove accounts.

**Comment by dilawfird** [comment]
I just commit a fix to keep the app alive it should be fixed i'm testing it right now.
